lib/vertica.rb

```diff
@@ -29,8 +30,8 @@ def self.quote(value)
       when nil        then 'NULL'
       when false      then 'FALSE'
       when true       then 'TRUE'
-      when DateTime   then value.strftime("'%Y-%m-%d %H:%M:%S'::timestamp")
-      when Time       then value.strftime("'%Y-%m-%d %H:%M:%S'::timestamp")
+      when DateTime   then value.strftime("'%Y-%m-%d %H:%M:%S %z'::timestamptz")
+      when Time       then value.strftime("'%Y-%m-%d %H:%M:%S %z'::timestamptz")
```

Comment:
Updated to use ISO format, and also include microseconds (the resolution Vertica supports).